images/linux/scripts/installers/lerna.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -e
```

Comment:
Use the nodejs.sh script to install lerna

Reply:
I suggest creating a new toolset section with Node.js modules as well